consul_config.py.ctmpl

```diff
@@ -0,0 +1,51 @@
+{{- define "KEY" -}}
+    {{ key (printf "docker-server-configs/AB/config.%s.json/%s" (env "DEPLOY_ENV") .) }}
+{{- end -}}
+
+{{- define "KEY_ARRAY" -}}
+    {{- range $index, $element := (key (printf "docker-server-configs/AB/config.%s.json/%s" (env "DEPLOY_ENV") .) | parseJSON) -}}
+        "{{.}}",
+    {{- end -}}
+{{- end -}}
+
+import os
+
+DEBUG = False
+SECRET_KEY = '''{{template "KEY" "secret_key"}}'''
+
+{{if service "acousticbrainz-pgbouncer"}}
+{{with index (service "pgbouncer-master") 0}}
+SQLALCHEMY_DATABASE_URI = "postgresql://acousticbrainz:acousticbrainz@{{.Address}}:{{.Port}}/acousticbrainz"
+POSTGRES_ADMIN_URI="postgresql://postgres@{{.Address}}:{{.Port}}/template1"
+POSTGRES_ADMIN_AB_URI="postgresql://postgres@{{.Address}}:{{.Port}}/acousticbrainz"
+{{end}}
+{{end}}
+
+{{if service "acousticbrainz-redis"}}
```

Comment:
AB's use of redis is not very great -- let's not duplicate this and use the main metabrainz redis since that is on track for no longer being a single point of failure.

Reply:
By metabrainz redis, do you mean this one: https://github.com/metabrainz/docker-server-configs/blob/master/scripts/nodes/hip.sh#L10

or one of the many started by musicbrainz on williams: https://github.com/metabrainz/docker-server-configs/blob/master/scripts/nodes/williams.sh#L18

Reply:
LOL. The same one that LB uses, should be fine.